modin/engines/base/frame/data.py

```diff
@@ -1644,13 +1645,15 @@ def broadcast_apply_full_axis(
                 The axis to apply over (0 - rows, 1 - columns).
             func : callable
                 The function to apply.
-            other : other Modin frame to broadcast
+            other : others Modin frames to broadcast
             new_index : list-like (optional)
                 The index of the result. We may know this in advance,
                 and if not provided it must be computed.
             new_columns : list-like (optional)
                 The columns of the result. We may know this in
                 advance, and if not provided it must be computed.
+            apply_indices : list of ints (optional),
```

Comment:
`apply_indices` is not a list of ints if we look at `apply_indices = list(agg_func.keys()) if isinstance(agg_func, dict) else None`, isn't it?

Reply:
Yeah, you're right.

`apply_indices` here is an array of labels of `axis^1`, fixed it in the doc string